tools/cldr-apps/js/src/specialToComponentMap.js

```diff
@@ -17,6 +18,7 @@ const specialToComponentMap = {
   about: AboutPanel,
   add_user: AddUser,
   auto_import: AutoImport,
+  lock_account: LockAccount,
```

Comment:
hooray, another modern component! 🎆 (Glad I made that the default for specials…)